addon/converter/fixture-converter.js

```diff
@@ -94,7 +94,7 @@ export default class {
   }
 
   transformRelationshipKey(relationship) {
-    let transformFn = this.getTransformKeyFunction(relationship.type, 'Relationship');
+    let transformFn = this.getTransformKeyFunction(relationship.parentType.modelName, 'Relationship');
```

Comment:
Actually the serializer of the parent of the relationship should be asked for the key of the given relationship. I am actually quite surprised that this hasn't surfaced earlier, so maybe I am missing something?

Reply:
Oddly I am using AMS and polymorphic relationships all over the place, so I would have thought so too.  Sometimes bugs just suddenly stand up and say 'notice me' .. or maybe you just got the perfect edge case here